spec/rails_helper.rb

```diff
@@ -6,6 +6,12 @@
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
 
+if ENV['RAILS_ENV'] == 'test'
+  require 'simplecov'
+  SimpleCov.start 'rails'
+  puts "required simplecov"
```

Comment:
Since this code works reliably, could we remove the `puts` line here?

Reply:
Sure.